file-upload/file-upload-java-sbt/fileupload-api/src/main/java/com/example/fileupload/api/FileUploadService.java

```diff
@@ -29,9 +30,7 @@ default Descriptor descriptor() {
                 .withCalls(
                         pathCall("/api/echo", this::uppercaseEcho)
                 )
-                .withServiceAcls(
-                        path("/api/echo"),
-                        path("/api/files")
-                );
+                .withAutoAcl(true)
+                .withServiceAcls(path("/api/files"));
```

Comment:
This ACL no longer exposes `"/api/echo"` via the gateway.

Reply:
have you tested it? I remember to have tested it and it was working. I may be wrong tough.

Reply:
You are right!

TIL https://github.com/lagom/lagom/blob/db77b62e66ba9ba725a25a3aaf09267b4e25037b/service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/Descriptor.java#L805